packages/account/tests/account.e2e.spec.ts

```diff
@@ -166,6 +178,50 @@ describe("Account Tests", () => {
     expect(newBalance - balance).toBe(1n);
   }, 60000);
 
+  it("Should throw and error if missing field for ERC20 Paymaster user op", async () => {
```

Comment:
does this mean buildUserOp would also throw?

Reply:
Yes, if ERC20 mode is passed without all required fields